etc/ProdOfflineConfiguration.py

```diff
@@ -674,14 +694,15 @@
                disk_node="T2_CH_CERN",
                scenario=ppScenario)
     
-DATASETS = ["ParkingLLP"]
+DATASETS = ["ParkingLLP0", "ParkingLLP1"]
```

Comment:
FTR, removing `ParkingLLP` and `ParkingHH` (in favour of `ParkingHH{0,1}` and `ParkingLLP{0,1}`) is okay with HLT, in the sense that today no 2025 menus contain the `ParkingLLP` and `ParkingHH` PDs.

I do not necessarily know whether or not it is okay to remove them in general from this config, that's to be determined by T0 and the other relevant experts.

PS. The same argument for removal applies to `EGamma`, `JetMET`, `DisplacedJet`, etc, etc.